be/src/olap/rowset/segment_v2/segment.cpp

```diff
@@ -48,9 +48,10 @@ Status Segment::open(io::FileSystem* fs, const std::string& path, uint32_t segme
 Segment::Segment(uint32_t segment_id, const TabletSchema* tablet_schema)
         : _segment_id(segment_id), _tablet_schema(*tablet_schema) {
 #ifndef BE_TEST
-    _mem_tracker = StorageEngine::instance()->tablet_mem_tracker();
+    _mem_tracker = MemTracker::create_virtual_tracker(
+            -1, "Segment", StorageEngine::instance()->tablet_mem_tracker());
 #else
-    _mem_tracker = MemTracker::get_process_tracker();
```

Comment:
should process_tracker be used in other places?

Reply:
`process_tracker` should not be consumed by manual consume/release,
Manual consumption in Segment should create a new virtual tracker
(I'm refactoring mem tracker to avoid this)